### PR TITLE
IDE-4805 add step to config proxy for blade and jpm on installers

### PR DESCRIPTION
**Pull request by yanan**
Added proxy for jpm.ini and blade.ini. Here are some screenshots:

screenshot1

screenshot2

screenshot3

### CODE REVIEW

File: build/installers/liferay-workspace-with-devstudio/liferay-workspace-with-devstudio.xml [hunk 1]
```diff
@@ -484,6 +485,83 @@ ${java_executable}
                         </runProgram>
                     </elseActionList>
                 </if>
+                <if>
+                    <explanation>need to configured Blade CLI with proxy settings</explanation>
```
Comment (gamerson) [inline build/installers/liferay-workspace-with-devstudio/liferay-workspace-with-devstudio.xml:489]:
```suggestion
                    <explanation>need to configure Blade CLI with proxy settings</explanation>
```

File: build/installers/liferay-workspace-with-devstudio/liferay-workspace-with-devstudio.xml [hunk 2]
```diff
@@ -804,6 +882,174 @@ ${java_executable}
                 </compareText>
             </ruleList>
         </fileParameter>
+        <choiceParameterGroup>
+            <name>bladeproxy</name>
+            <description>Config proxy for JPM and Blade CLI</description>
+            <explanation>If you have proxy server requirements and want to use Blade CLI, you must configure your proxy for it using JPM. And please note that Blade CLI’s proxy settings are sometimes reset when executing blade update.</explanation>
```
Comment (gamerson) [inline build/installers/liferay-workspace-with-devstudio/liferay-workspace-with-devstudio.xml:888]:
```suggestion
            <explanation>If your workstation requires a proxy server, you must configure it for use by Blade by specifying it here.</explanation>
```

File: build/installers/liferay-workspace/liferay-workspace.xml [hunk 3]
```diff
@@ -229,18 +234,111 @@
...
+            <file>../components/set-installer-variable-gw.xml</file>
+        </include>
+        <include>
+            <file>../components/set-installer-variable-bnd.xml</file>
+        </include>
+        <include>
+            <file>../components/run-jpm-install-blade.xml</file>
+        </include>
+        <if>
+            <explanation>need to configured Blade CLI with proxy settings</explanation>
```
Comment (gamerson) [inline build/installers/liferay-workspace/liferay-workspace.xml:260]:
```suggestion
            <explanation>need to configure Blade CLI with proxy settings</explanation>
```

File: build/installers/liferay-workspace/liferay-workspace.xml [hunk 4]
```diff
@@ -910,5 +1015,169 @@
                 </ruleGroup>
             </ruleList>
         </parameterGroup>
+        <choiceParameterGroup>
+            <name>bladeproxy</name>
+            <description>Config proxy for JPM and Blade CLI</description>
+            <explanation>If you have proxy server requirements and want to use Blade CLI, you must configure your proxy for it using JPM. And please note that Blade CLI’s proxy settings are sometimes reset when executing blade update.</explanation>
```
Comment (gamerson) [inline build/installers/liferay-workspace/liferay-workspace.xml:1021]:
```suggestion
            <explanation>If your workstation requires a proxy server, you must configure it for use by Blade by specifying it here.</explanation>
```

**Comment by yanan** [comment]
Thanks for the updates, I will send a new pull requst.

**Pull request closed without merging**